### ISSUE TITLE
[Internet Archive] The app crashes when clicking the Copy link option from context menu triggered for a opened video

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#3331]
**[Affected Versions]:**
- Firefox Reality version 10 RC4 (2999129).

**[Affected Platforms]:**
- Oculus Quest - Android 7.1.1
- HTC Vive Focus - Android 7.1.2
- Pico Neo2 - Android 8.1.0

**[Prerequisites]:**
- The Firefox Reality browser is open.

**[Steps to reproduce]:**
1. Navigate to https://archive.org/details/opensource_movies?and[]=mediatype%3A%22movies%22 page.
2. Open a video but do not play it.
3. Long click in the video area.
4. Click the "Copy link" button from the Context Menu displayed.
5. Observe the behavior.

**[Expected results]:**
- The link is copied or the Context Menu should not be displayed for an opened video.

**[Actual results]:**
- The app crashes.

**[Notes]:**
- It seems that the link for the video is not displayed in the Context menu, above the Copy link button.
- I was not able to reproduce this issue on websites like Youtube or Vimeo.
- If the video is played the "Download Video" and "Copy link" options are displayed in the context menu and they work as expected.
- Attached are the screen recording, crash report along with the logcat for the issue:
	- Screen recording: [link](https://drive.google.com/open?id=1ybA2ceacb8WpuBS0hCE_OtEEOHKSgLF0).
	- Crash Report: [link](https://crash-stats.mozilla.org/report/index/d4bf5ed9-026f-4304-9653-341f20200505).
	- Logcat: [link](https://drive.google.com/open?id=1bySrpG9Tp4jetfXzqgGbK2Hiiuw8N3X8).

**Comment by bluemarvin** [comment]
Fixed by #3332

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and it's no longer reproducible with the Firefox Reality version 10 RC5 (e84d3c9). The context menu is no longer triggered for a opened video on https://archive.org/ website.

Tested on HTC Vive Focus (Android 7.1.2), HTC Vive Focus Plus (Android 7.1.2), Oculus Quest (Android 7.1.1), Oculus Go (Android 7.1.2), Pico G2 4K (Android 8.1.0) and Pico Neo 2 (Android - 8.1.0) devices.
